packages/console/src/assets/images/social-connectors-placeholder.svg

```diff
@@ -0,0 +1,75 @@
+<path d="M180.51 38.6834L180.245 39.1782L197.43 48.3703L197.695 47.8755L180.51 38.6834Z" fill="#3F3D56"/>
+<path d="M215.567 43.2882L198.384 52.4849L198.649 52.9796L215.831 43.783L215.567 43.2882Z" fill="#3F3D56"/>
+<path d="M144.625 24.7583L146.871 22.9617C145.126 22.7692 144.409 23.7209 144.116 24.4741C142.753 23.908 141.268 24.6499 141.268 24.6499L145.763 26.2815C145.536 25.6759 145.141 25.1475 144.625 24.7583Z" fill="#3F3D56"/>
+<path d="M89.7236 8.8979L91.9699 7.10133C90.2248 6.90881 89.5079 7.8605 89.2144 8.61377C87.8512 8.04768 86.3671 8.78957 86.3671 8.78957L90.8614 10.4212C90.6347 9.81557 90.24 9.28716 89.7236 8.8979Z" fill="#3F3D56"/>
+<path d="M179.274 1.82173L181.52 0.0251599C179.775 -0.167361 179.058 0.784328 178.765 1.5376C177.401 0.971505 175.917 1.7134 175.917 1.7134L180.412 3.345C180.185 2.7394 179.79 2.21099 179.274 1.82173Z" fill="#3F3D56"/>
+<path d="M183.809 106.643C183.212 106.643 182.625 106.801 182.108 107.1C181.584 106.625 180.932 106.313 180.233 106.201C179.534 106.089 178.818 106.183 178.171 106.471C177.524 106.758 176.975 107.227 176.59 107.821C176.205 108.415 176 109.108 176.001 109.815H187.213C187.152 108.954 186.767 108.148 186.135 107.56C185.504 106.971 184.672 106.644 183.809 106.643Z" fill="#3F3D56"/>
+<path d="M69.8581 106.643C69.261 106.643 68.6744 106.801 68.1575 107.1C67.6326 106.625 66.9811 106.313 66.2822 106.201C65.5833 106.089 64.867 106.183 64.2202 106.471C63.5735 106.758 63.0242 107.227 62.6389 107.821C62.2537 108.415 62.0491 109.108 62.0499 109.815H73.2619C73.2008 108.954 72.8157 108.148 72.1841 107.56C71.5526 106.971 70.7214 106.644 69.8581 106.643Z" fill="#3F3D56"/>
+<path d="M128.176 106.643C127.579 106.643 126.992 106.801 126.475 107.1C125.95 106.625 125.299 106.313 124.6 106.201C123.901 106.089 123.184 106.183 122.538 106.471C121.891 106.758 121.342 107.227 120.956 107.821C120.571 108.415 120.367 109.108 120.367 109.815H131.579C131.518 108.954 131.133 108.148 130.502 107.56C129.87 106.971 129.039 106.644 128.176 106.643Z" fill="#3F3D56"/>
+<path d="M115.87 98.6343H83.5226V99.6776H88.5674V109.591H89.6108V99.6776H109.26V109.591H110.303V99.6776H115.87V98.6343Z" fill="#3F3D56"/>
+<path d="M115.934 95.5122H83.5862V96.5557H115.934V95.5122Z" fill="#3F3D56"/>
+<path d="M115.934 92.9038H83.5862V93.9473H115.934V92.9038Z" fill="#3F3D56"/>
+<path d="M115.934 90.2949H83.5862V91.3384H115.934V90.2949Z" fill="#3F3D56"/>
+<path d="M166.379 98.6343H134.032V99.6776H139.077V109.591H140.12V99.6776H159.769V109.591H160.813V99.6776H166.379V98.6343Z" fill="#3F3D56"/>
+<path d="M166.443 95.5122H134.095V96.5557H166.443V95.5122Z" fill="#3F3D56"/>
+<path d="M166.443 92.9038H134.095V93.9473H166.443V92.9038Z" fill="#3F3D56"/>
+<path d="M166.443 90.2949H134.095V91.3384H166.443V90.2949Z" fill="#3F3D56"/>
+</g>
+<defs>
+<clipPath id="clip0_1640_62321">
+<rect width="243.484" height="110" fill="white" transform="translate(0.757812)"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
Better format this file to at least have better indents

Reply:
this file is exported from Figma and is not supposed to be modified

Reply:
I understand, but still better off format it with IDE's auto format feature or something. Would improve the overall readability of the svg files.

Reply:
VSCode won't format svg?